### ISSUE TITLE
Pause playback when a remote computer connects to the PC using AnyDesk

### ISSUE THREAD

**Issue by ASchneiderBR** [rocksdanister/lively#1289]
**Is your feature request related to a problem? Please describe.**
Hello. First of all, thanks for creating this great app, I just love it! I was wondering if it would be possible to automatically stop Lively playback when a remote computer connects to the machine using AnyDesk. Having to render the effects using their codec is very resource and bandwidth consuming. Thanks.

**Describe the solution you'd like**
I think you already have a solution the default windows remote desktop connects, would be nice to have it working the same way for AnyDesk (maybe TeamViewer as well?)

Thank you very much!

**Comment by rocksdanister** [comment]
Related https://github.com/rocksdanister/lively/issues/207

**Comment by ASchneiderBR** [comment]
Hello. It did not worked here when I've connected to my PC, running Lively (fluid) using AnyDesk. The animation was still being rendered after the connection was established. Thanks!

**Comment by rocksdanister** [comment]
Yes, only Windows RDP session detection code is written. I just added that issue for reference.

**Comment by ASchneiderBR** [comment]
Got it, thanks. So there's no plan for having this enhancement in the future? Or should I just close this feature request? Thanks!

**Comment by rocksdanister** [comment]
I don't have a release time for this personally but keep it open so that others can help.

**Comment by rocksdanister** [comment]
Alternatives you can try for now:
1. Pause tray-menu:
<img width="531" alt="image" src="https://user-images.githubusercontent.com/17554161/177207673-d35613f9-73c3-4dc8-a41d-ef0221958acc.png">

2. Run a custom script that runs during session (windows task sheduler, shortcut, ahk etc):
https://github.com/rocksdanister/lively/wiki/Command-Line-Controls

3. Try adding pause rule (I doubt this will work unless the app is in focus.)
<img width="513" alt="image" src="https://user-images.githubusercontent.com/17554161/177207790-917bee9a-cc19-45fc-9a3d-098e65066e49.png">

**Comment by ASchneiderBR** [comment]
Perfect, I'll keep it open then. Thanks again, have a great day.
